src/components/modules/banners/BannerTypes.ts

```diff
@@ -8,14 +8,18 @@ export type BannerTargeting = {
     isPaidContent?: boolean;
     showSupportMessaging: boolean;
     engagementBannerLastClosedAt?: string;
+    subscriptionsBannerLastClosedAt?: string;
     mvtId: number;
     countryCode: string;
+    switches: {
+        remoteSubscriptionsBanner: boolean;
+    };
 };
 
 export type BannerTestTracking = {
     abTestName: string;
-    abTestVariant: string;
-    campaignCode: string;
+    abTestVariant: string | null;
+    campaignCode: string | null;
 };
```

Comment:
Can we remove the `| null`  and restore these to `string`?

Reply:
Yes! Thanks George 👍